test/functional/includeconf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
These don't actually need to be separate tests. You can just have the following in `setup_chain()`:

```python
        with open(os.path.join(self.options.tmpdir + "/node0", "relative.conf"), "w", encoding="utf8") as f:
            f.write("uacomment=relative\nincludeconf=relativeofrelative.conf\n")
        with open(os.path.join(self.options.tmpdir + "/node0", "bitcoin.conf"), 'a', encoding='utf8') as f:
            f.write("uacomment=main\nincludeconf=relative.conf\n")
        with open(os.path.join(self.options.tmpdir + "/node0", "commandline.conf"), "w", encoding="utf8") as f:
            f.write("uacomment=commandline\n")
        with open(os.path.join(self.options.tmpdir + "/node0", "relativeofrelative.conf"), "w", encoding="utf8") as f:
            f.write("uacomment=relativeofrelative\n")
```

and the following line in `__init__()`:

```python
        self.extra_args = [['-includeconf=commandline.conf']]
```

and then keep the same assert:

```python
        assert subversion.endswith("main; relative)/")
```

to check that neither `comandline.conf` nor `relativeofrelative.conf` have been included.

(I recommend you also update the docstring to describe what's happening)